packages/babel-traverse/src/path/conversion.ts

```diff
@@ -397,63 +399,96 @@ function hoistFunctionEnvironment(
   return { thisBinding, fnPath };
 }
 
-function standardizeSuperProperty(superProp) {
+type LogicalOp = Parameters<typeof logicalExpression>[0];
+type BinaryOp = Parameters<typeof binaryExpression>[0];
+
+function isLogicalOp(op: string): op is LogicalOp {
+  return LOGICAL_OPERATORS.includes(op);
```

Comment:
If TS allows `const` and `type` with the same name, we could even export `type LOGICAL_OPERATORS = "||" | "&&" | "??"` from `@babel/types`.